tests/unit/core/TestInstallAddon.js

```diff
@@ -110,10 +110,10 @@ describe('withInstallHelpers inner functions', () => {
         dispatch, { _addonManager: getFakeAddonManagerWrapper(), guid, installURL });
       return setCurrentStatus()
         .then(() => {
-          expect(dispatch.calledWith({
-            type: INSTALL_STATE,
-            payload: { guid, status: ENABLED, url: installURL },
-          })).toBeTruthy();
+          sinon.assert.calledWith(
```

Comment:
I get that this will throw if it doesn't work but it feels a bit weird to not use our usual test API. Will this give us a better sense of the arg mismatch than `expect()`? If so I am all for it, otherwise it feels a bit weird.